NSS-Day-Cohort-66/rare-api-golden-droids
# Added basic authentication

Added `login_user` to `auth.py` and then imported it into the `__init__.py`. 
Added `login` to the `urlpatterns` in `urls.py`

## To Test
1. Pull down the `ts-basic-authentication` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Change method to POST for `http://localhost:8000/login`, in the body add an object with the following keys:
```
{
  "username": "Carrie1945",
  "password": "me"
}
```
4. Once you send your POST you should receive an object with the following keys: 
```
{
    "valid": true,
    "token": "fa2eba9be8282d595c997ee5cd49f2ed31f65bed"
}
```
7. Be sure you receive a 200 status
